pkg/cmd/pr/merge/merge.go

```diff
@@ -302,6 +302,8 @@ func (m *mergeContext) merge() error {
 				return cmdutil.FlagErrorf("--merge, --rebase, or --squash required when not running interactively")
 			}
 
+			m.infof("Merging pull request #%d (%s)\n", m.pr.Number, m.pr.Title)
```

Comment:
The linter complaining that the error is unchecked, given we were swallowing them above I'd say we're do the same here:

```suggestion
			_ = m.infof("Merging pull request #%d (%s)\n", m.pr.Number, m.pr.Title)
```